sagemaker_edge_manager/sagemaker_edge_example/sagemaker_edge_greengrass_example.ipynb

```diff
@@ -0,0 +1,1202 @@
+{
```

Comment:
<p>Instead of using <code>documentation</code>  for the hyperlink text, use the doc page name - <code>What is AWS IoT Greengrass?</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>